src/main/groovy/nextflow/Const.groovy

```diff
@@ -59,12 +59,12 @@ class Const {
     /**
      * The app build time as linux/unix timestamp
      */
-    static public final long APP_TIMESTAMP = 1529171346935
+    static public final long APP_TIMESTAMP = 1531207368095
 
     /**
      * The app build number
      */
-    static public final int APP_BUILDNUM = 4867
+    static public final int APP_BUILDNUM = 4873
 
```

Comment:
Please do not commit changes in this file since it's automatically modified.

Reply:
arghh... managed not to commit it the first time around